client/app/telecom/telephony/alias/portability/order/translations/Messages_fr_FR.xml

```diff
@@ -16,7 +16,9 @@
    <translation id="telephony_alias_portability_order_callNumber" qtlid="460756">Numéro à porter (format international)</translation>
    <translation id="telephony_alias_portability_order_number_international" qtlid="460769">N° au format international</translation>
    <translation id="telephony_alias_portability_order_callNumber_pattern_error" qtlid="460782">Veuillez saisir un numéro au format international (0033… ou +33…).</translation>
-   <translation id="telephony_alias_portability_order_desireDate" qtlid="392255">Date de portabilité souhaitée</translation>
+   <translation id="telephony_alias_portability_order_desireDate" qtlid="392255">Date de portabilité souhaitée :</translation>
```

Comment:
can you remove `qtlid` plz. To be sure that translation won't be overrided by translation script